packages/data-stores/src/help-center/actions.ts

```
@@ -148,27 +148,25 @@ export const setShowHelpCenter = function* (
 	}
 
 	if ( ! isE2ETest() ) {
-		try {
-			if ( canAccessWpcomApis() ) {
-				// Use the promise version to do that action without waiting for the result.
-				wpcomRequestPromise( {
-					path: '/me/preferences',
-					apiNamespace: 'wpcom/v2',
-					method: 'PUT',
-					body: {
-						calypso_preferences: { help_center_open: show },
-					},
-				} );
-			} else {
-				// Use the promise version to do that action without waiting for the result.
-				apiFetchPromise( {
-					global: true,
-					path: '/help-center/open-state',
-					method: 'PUT',
-					data: { help_center_open: show },
-				} as APIFetchOptions );
-			}
-		} catch {}
+		if ( canAccessWpcomApis() ) {
+			// Use the promise version to do that action without waiting for the result.
+			wpcomRequestPromise( {
+				path: '/me/preferences',
+				apiNamespace: 'wpcom/v2',
+				method: 'PUT',
+				body: {
+					calypso_preferences: { help_center_open: show },
+				},
+			} ).catch( () => {} );
+		} else {
+			// Use the promise version to do that action without waiting for the result.
+			apiFetchPromise( {
+				global: true,
+				path: '/help-center/open-state',
+				method: 'PUT',
+				data: { help_center_open: show },
+			} as APIFetchOptions ).catch( () => {} );
+		}
```

Since we were not awaiting these promises, the `try/catch` block here had no effect. Now we're silencing these errors. 

cc @alshakero